packages/manager/modules/carrier-sip/src/endpoints/endpoints.html

```diff
@@ -0,0 +1,30 @@
+<section>
+    <h3 data-translate="carrier_sip_endpoints"></h3>
```

Comment:
No `h2` above ?

Reply:
In fact here we ceate a dedicated `<section>` element but the page where this component lives contains a `<oui-page-header>` element with both property `heading` and `description` that result in `h1 + h2` element.

That's why we have decided to put a `h3` tag.

Is that make sens for you?

Thanks!

Reply:
This is the answer I was expecting 😄